src/collector/core/endpoints.py

```diff
@@ -0,0 +1,201 @@
+import os
```

Comment:
in the latest version of SKALE Proxy we have REST API to get all endpoints for all chains, so this entire module can be replaced with an API call to proxy. 

This will allow us to avoid code duplication & simplify architecture of stats collector